model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -361,9 +367,14 @@ async def generate_with_vllm(
     top_p,
     prompts,
     bar,
+    use_tool,
 ) -> List[CompletionOutput]:  # pragma: no cover
     from vllm import SamplingParams
 
+    model = (await engine.get_model_config()).model
```

Comment:
Does this have a risk of increasing the latency?

Reply:
it'll ask Ray workers to get model config (for some reason) and it should be pretty fast. also this call runs once for the whole batch instead of each prompt